news-functions/Handlers/GetStorageConnection.cs

```diff
@@ -28,50 +31,87 @@ public static class GetStorageConnection
                 SharedAccessExpiryTime = DateTime.UtcNow.AddDays(1),
                 Permissions = SharedAccessTablePermissions.Query
             };
+            log.LogInformation("created sas policy");
 
             var storageAccount = CloudStorageAccount.Parse(EnvironmentHelper.GetEnv("AccountStorage-Conn"));
             var tableClient = storageAccount.CreateCloudTableClient() ??
                               throw new Exception(nameof(storageAccount.CreateCloudTableClient));
-            var table = tableClient.GetTableReference("news") ??
-                        throw new Exception(nameof(tableClient.GetTableReference));
+            log.LogInformation($"Connected to storage account");
```

Comment:
This is DBG information, not worth anything to an operator under standard logging settings